src/main/java/com/arpnetworking/metrics/mad/parsers/JsonToRecordParser.java

```diff
@@ -289,32 +249,62 @@ public Record parse(final byte[] data) throws ParsingException {
         putVariablesVersion2fSteno(data.getCounters(), MetricType.COUNTER, variables);
         putVariablesVersion2fSteno(data.getGauges(), MetricType.GAUGE, variables);
 
-        // NOTE: We are mapping from the underscore prefixed annotations to non-prefixed ones.
-        // The prefixed annotations will overwrite the non-prefixed ones.
-        final ImmutableMap.Builder<String, String> mappedAnnotations = ImmutableMap.builder();
-        mappedAnnotations.put(Key.HOST_DIMENSION_KEY, context.getHost());
-        for (final Map.Entry<String, String> entry : annotations.getOtherAnnotations().entrySet()) {
-            final String key = entry.getKey();
-            final String value = entry.getValue();
-            if (PREFIXED_SERVICE_KEY.endsWith(key)) {
-                mappedAnnotations.put(Key.SERVICE_DIMENSION_KEY, value);
-            } else if (PREFIXED_CLUSTER_KEY.endsWith(key)) {
-                mappedAnnotations.put(Key.CLUSTER_DIMENSION_KEY, value);
-            } else if (!Key.HOST_DIMENSION_KEY.equals(key)
-                    && !Key.SERVICE_DIMENSION_KEY.endsWith(key)
-                    && !Key.CLUSTER_DIMENSION_KEY.endsWith(key)) {
-                mappedAnnotations.put(key, value);
-            }
-        }
+        final ImmutableMap.Builder<String, String> annotationsBuilder = ImmutableMap.builder();
+        annotationsBuilder.putAll(annotations.getOtherAnnotations());
+        annotationsBuilder.put(PREFIXED_HOST_KEY, context.getHost());
```

Comment:
In the other formats you effectively removed host, service and cluster but for f you kept host only. Why the Difference?